orderer/consensus/hlmirbft/chain.go

```diff
@@ -600,12 +679,23 @@ func (c *Chain) writeBlock(block *common.Block, index uint64) {
 func (c *Chain) ordered(msg *orderer.SubmitRequest) (err error) {
 
 	seq := c.support.Sequence()
-
 	if msg.LastValidationSeq < seq {
 
 		if c.isConfig(msg.Payload) {
```

Comment:
You'll need to merge main into this branch and then sort out the conflicts. I've changed the name of this function, `ordered`, to `checkMsg`.